docs/source/creating_an_image_processing_pipeline/index.rst

```diff
@@ -142,6 +148,88 @@ Finding and Decoding Spots
+starfish requires manually selecting a global threshold value in :py:class:`.ThresholdBinarize`.
+
+When overlapping cells are labeled as one connected component, they are typically segmented by
+using a `distance transformation followed by the watershed algorithm <https://www.mathworks
+.com/company/newsletters/articles/the-watershed-transform-strategies-for-image-segmentation
+.html>`_. Watershed is a classic image processing algorithm for separating objects in images and
+can be applied to all types of images. Pairing it with a distance transform is particularly
+useful for segmenting convex shapes like cells.
+
+A segmentation pipeline that consists of thresholding, connected component analysis, and watershed
+is the simplest and fastest to implement but its accuracy is highly dependent on image quality.
+The signal-to-noise ratio of the cell stain must be high enough for minimal errors after
+thresholding and binary operations. And the nuclei or cell shapes must be convex to meet the
+assumptions of the distance transform or else it will over-segment. Starfish includes the basic
+functions to build a watershed segmentation pipeline and a predefined :py:class:`.Watershed`
+segmentation class that uses the :term:`primary images<Primary Images>` as the cell stain.
+
+* Tutorial: :ref:`Ways to segment by thresholding and watershed<tutorial_watershed_segmentation>`
+
+Manually Defining Cells
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The most accurate but time-consuming approach is to manually segment images using a tool such as
+`ROI manager <https://imagej.net/docs/guide/146-30.html#fig:The-ROI-Manager>`_ in FIJI (ImageJ). It
+is a straightforward process that starfish supports by allowing ROI set binaries to be imported as a
+:py:class:`.BinaryMaskCollection`. These masks can then be integrated into the pipeline for
+visualization and assigning spots to cells.
+
+* Tutorial: :ref:`Loading ImageJ ROI set<tutorial_manual_segmentation>`
+
+Machine-Learning Methods
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Besides the two classic cell segmentation approaches mentioned above, there are machine-learning
+methods that aim to replicate the accuracy of manual cell segmentation while reducing the labor
+required. Machine-learning algorithms for segmentation are continually improving but there is no
+perfect solution for all image types yet. These methods require training data (e.g. stained
+images with manually defined labels) to train a model to predict cell or nuclei locations in test
+data. There are `exceptions that don't require training on your specific data <http://www.cellpose
+.org/>`_ but generally training the model is something to consider when evaluating how much time
+each segmentation approach will require.
+
+Starfish currently has built-in functionality to support `ilastik <https://www.ilastik.org/>`_, a
+segmentation toolkit that leverages machine-learning. Ilastik has a Pixel Classification
```

Comment:
Is there a link to Ilastik's pixel classification workflows that might be pertinent here?

Reply:
Yes added